docs/hello.adoc

```diff
@@ -87,10 +89,10 @@ NOTE: Try clicking other content items in Content Studio, and you should simply
 The responsiblity of HelloProcessor is to produce the props required by a component.
 A processor may also intercept the request, for instance by producing a redirect.
```

Comment:
We changed it so it can not return the response any more. Replace it with the definition of DataFetcher:
`DataFetcher is a registry of data fetching functions for components by their type. It is used in the backend to get props for rendering component views from ComponentRegistry.`